modules/gdscript/tests/scripts/parser/features/export_variable.gd

```diff
@@ -1,18 +1,29 @@
-@export var example = 99
-@export_range(0, 100) var example_range = 100
-@export_range(0, 100, 1) var example_range_step = 101
-@export_range(0, 100, 1, "or_greater") var example_range_step_or_greater = 102
-
-@export var color: Color
-@export_color_no_alpha var color_no_alpha: Color
-@export_node_path("Sprite2D", "Sprite3D", "Control", "Node") var nodepath := ^"hello"
+const Utils = preload("../../utils.notest.gd")
 
+@export_storage var test_storage
+@export_storage var test_storage_weak_int = 1 # Property info still `Variant`, unlike `@export`.
+@export_storage var test_storage_hard_int: int = 2
```

Comment:
Nitpick, but why do we test them first? I'd go from most common and generic to least common. `@export` is the most basic one, so IMO it should remain first.